CHANGELOG.md

```diff
@@ -6,8 +6,23 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 2.10.1
 ------
 
+**ENHANCEMENTS**
+
+- EFA kernel module now installed also on ARM instances with `alinux2` and `ubuntu1804`
+
 **CHANGES**
 - Use inclusive language in recipe names and internal naming convention.
+- Upgrade EFA installer to version 1.11.0
```

Comment:
I wouldn't include EFA changelog in ours but only mention the version of the main libraries

Reply:
It seems to me that we did the same was done for 2.10.0. I can change it BTW.